vendor/brave-ios/Ledger/Generated/Enums.h

```diff
@@ -29,22 +13,6 @@ typedef NS_ENUM(NSInteger, BATLogLevel) {
   BATLogLevelLogResponse = 6
 } NS_SWIFT_NAME(LogLevel);
 
-typedef NS_ENUM(NSInteger, BATPublisherExclude) {
-  BATPublisherExcludeAll = -1,
-  BATPublisherExcludeDefault = 0,
-  BATPublisherExcludeExcluded = 1,
-  BATPublisherExcludeIncluded = 2
-} NS_SWIFT_NAME(PublisherExclude);
-
-typedef NS_ENUM(NSInteger, BATReportType) {
-  BATReportTypeGrant = 0,
-  BATReportTypeAutoContribution = 1,
-  BATReportTypeDeposit = 2,
-  BATReportTypeAds = 3,
-  BATReportTypeTipRecurring = 4,
-  BATReportTypeTip = 5
-} NS_SWIFT_NAME(ReportType);
-
 typedef NS_ENUM(NSInteger, BATURLMethod) {
```

Comment:
Looks like this enum should be removed too, but the new one may be improperly named based on generation (Most likely `BATUrlMethod`)

Reply:
removed